design/open/oidc-support.md

```diff
@@ -0,0 +1,22 @@
+# OIDC support in lakeFS
+
+As suggested by @arielshaqed, we started researching the option to implement OIDC support as part of lakeFS.
+
+In such a solution, the person who configures lakeFS will supply the parameters required for implementing OIDC, including the provider's domain, the client ID and secret, and a callback URL. API access will still use credentials managed internally like today.
+
+As a first step, authorization will still be managed internally by lakeFS.
+
+## Requirements
+
+
+### lakeFS admin
+- As part of the lakeFS configuration file, you can enable an integration with an OIDC provider
+- The admin can configure a global default lakeFS IAM groups to create all users with.
+- Using the "policy" claim on the ID token, the admin can specify an initial group for specific user, to replace the global default group.
```

Comment:
Could you explain a bit more about how the admin configures this, maybe even for a specific provider?  Will this require admins to control groups on the OIDC provider, or on lakeFS, or on both???

Also, users should be able to belong to more than one group; not sure I understand how that will work.